notification-portlet-webapp/src/main/resources/datasource.properties

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to Jasig under one or more contributor license
+# agreements. See the NOTICE file distributed with this work
+# for additional information regarding copyright ownership.
+# Jasig licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a
+# copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+##### Default HSQL Configuration
+hibernate.connection.driver_class=org.hsqldb.jdbc.JDBCDriver
+hibernate.connection.url=jdbc:hsqldb:mem:notice
+hibernate.connection.username=sa
+hibernate.connection.password=
+hibernate.connection.validationQuery=select 1 from INFORMATION_SCHEMA.SYSTEM_USERS
+hibernate.dialect=org.hibernate.dialect.HSQLDialect
```

Comment:
@chasegawa -- Absolutely.  But this approach with Maven filters is something we do with _bundled_ portlets in uportal-portlets-overlay.  When we update the version pulled into uPortal, we should definitely add these.